.github/workflows/test.yml

```diff
@@ -268,6 +268,6 @@ jobs:
             go.mod
             go.sum
       - name: Test
-        run: go test -test.timeout 30m -failfast -v github.com/celestiaorg/celestia-app/e2e/qgb
+        run: go test -test.timeout 60m -failfast -v github.com/celestiaorg/celestia-app/e2e/qgb
```

Comment:
Check the following run of e2e:
https://github.com/celestiaorg/celestia-app/runs/7031106876?check_suite_focus=true
It took 35min without the full behavior test (this one is so heavy I disabled it).
The block time is set to 1 sec. I think currently the issue why the tests are taking so long, and the reason that we have to wait for 20 blocks, 30 blocks, and more in some tests, is due to the fact that the `celestia-app` binary take time to run. This will need to be investigated to see why it takes 30sec or more to execute even the smallest commands... 
When that issue is fixed,  then we can readjust the e2e to wait for less time.

Reply:
ahh I see